### ISSUE TITLE
Change stale bot to not close issues with certain label

### ISSUE THREAD

**Issue by jonnylangefeld** [argoproj/argo-workflows#3517]
# Summary

As I was working with the software, I found several issues that were relevant to what I was doing, that never had a resolution but were closed by the stale bot. 

# Motivation

This software is gaining traction and issues are raised more rapidly than they can be resolved.

# Proposal

I propose to introduce another label, that can be added to an issue, that might be a good feature, but the priority is too low to work on it until the stale bot decides to close it. If this additional label is set, the stale bot doesn't close it.

---

**Message from the maintainers**:

If you wish to see this enhancement implemented please add a 👍 reaction to this issue! We often sort issues this way to know what to prioritize.

**Comment by alexec** [comment]
Anything labelled "enhancement" or "security" will not be closed. Further more, stable bot re-opens issues when there is a reply. Stale bot only closes issues where the author is not longer replying to messages. 

Did you managed to find solutions to your problems? Could you link to them?

**Comment by jonnylangefeld** [comment]
That's good, I wasn't aware of the "enhancement" label exclusion. I think that should be sufficient.
I was looking to pass complex parameters to a template. I searched for "[json parameter](https://github.com/argoproj/argo/search?q=json+parameter&type=Issues)". I feel like there were issues opened, but it's not possible currently right?
I found issues like [this](https://github.com/argoproj/argo/issues/1635) that had a similar request, but you're right, there was no enhancement label and the author didn't reply anymore.
